Builder/Exception/MissingAssociationAdmin.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Builder\Exception;
+
+/**
+ * Signals that a field description is missing an association admin when a form type requires it.
+ *
+ * @author ju1ius <ju1ius@laposte.net>
+ */
+final class MissingAssociationAdmin extends \LogicException
```

Comment:
> This is not useless at all.

> With this suffix, you have a quick view of what kind of class you have here.


Related video : https://vimeo.com/album/2661665/video/74316116 (53:30)

Transcript : 

> How do you not know that something is an exception ? Where does this exception occurs ? In a throws clause, […] in a catch statement, in an inheritance chain when you create a new one. In other words you know damn well it's an exception…

So IMO it is very much useless, and the only reason to do this is to follow the Symfony standards, so let's not pretend it is anything else.